### ISSUE TITLE
Migrate wiki to github

### ISSUE THREAD

**Issue by xizhao** [fossology/fossology#15]
It would nice to be able to contribute edits to a wiki on github as well.  i.e. creating an entry on building nomos on mac:

`brew update`
`brew install glib`

Then edit `makefile.conf` to append brew's include path to glib as a CFLAG:

`-I/usr/local/Cellar/glib/2.42.0/include/glib-2.0/`

**Comment by bufferoverflow** [comment]
I think...
* We should keep the "old" wiki as archive and put it in read-only mode.
* Add most relevant stuff to CONTRIBUTING.md, README.md) and co within the source tree
* delete old unused information from wiki's and focus on the source tree

best!
-roger

**Comment by silverhook** [comment]
Is it still the plan to migrate and consolidate the documentation onto GitHub Wiki as stated in this bug and bug #97?

If so, I would be interested to help out a bit with.

**Comment by bobgob** [comment]
Hello Matija,

I am no longer involved with FOSSology but if no one replies, you could contact michael.c.jaeger@siemens.com <mailto:michael.c.jaeger@siemens.com>.  Michael has been running the project since I left a year ago.

Good luck,
Bob Gobeille

**Comment by silverhook** [comment]
Dear Bob,

I realise that @mcjaeger took over the project about a year ago (sad to see you go, BTW). This is exactly why I’m asking if this is still the plan.

Currently the documentation seems to be scattered around a bit and I would be be willing to help clean it up.

**Comment by mcjaeger** [comment]
Oh, actually good practice that you wrote. Problem with the git hub wiki is that most of the redmine content is impossible to migrate to the github so this stuck, but if you look at the wiki f the github project pages, most of the content is there.
We have had the idea for the installation / administration / user content to go to wiki.fossology.org.

**Comment by silverhook** [comment]
Right, so the idea is now to (eventually) move all relevant documentation to http://wiki.fossolog.org (DokuWiki)?

**Comment by kestewart** [comment]
Hi Matija,    Your help is most welcome.   Michael and others have been moving the developer documentation to github along with the code, but the user documentation (how to install it, use it, etc. ) could definitely use some help. 
see: https://wiki.fossology.org/  Which is where there's currrent talk of centralizing it.
Kate 

 

    On Monday, October 17, 2016 10:36 AM, Matija Šuklje <notifications@github.com> wrote:
 

 Dear Bob,I realise that @mcjaeger took over the project about a year ago (sad to see you go, BTW). This is exactly why I’m asking if this is still the plan.Currently the documentation seems to be scattered around a bit and I would be be willing to help clean it up.—

**Comment by silverhook** [comment]
Thanks @kestewart, I’ll look into it :)

**Comment by mcjaeger** [comment]
please consider also that the main issue with the migration was not so much which wiki, but that tables like 

http://archive15.fossology.org/projects/fossology/wiki/Nomos_Test_Cases

Do not go well into the wiki of github. Actually the wiki of integration test case descriptions is the worst issue I have with the migration. 

I haven't looked into a markup converter from redmine to docuwiki (wiki.fossology.org) or markdown (github), maybe that could be of great help here.

**Comment by silverhook** [comment]
For simple tables, it should be pretty easy:

You can get this:

| Left align | Right align | Center align |
|:-----------|------------:|:------------:|
| This       |        This |     This     
| column     |      column |    column    
| will       |        will |     will     
| be         |          be |      be      
| left       |       right |    center    
| aligned    |     aligned |   aligned

with this code:

~~~~.markdown
| Left align | Right align | Center align |
|:-----------|------------:|:------------:|
| This       |        This |     This     
| column     |      column |    column    
| will       |        will |     will     
| be         |          be |      be      
| left       |       right |    center    
| aligned    |     aligned |   aligned
~~~~

MarkDown is limited (as any simplistic markup), but the table extension is not uncommon and you can also just dump raw HTML into it if needed.

For converting, I usually first look if [Pandoc](http://pandoc.org/) can handle it – and both Textile and Markdown are covered (unsure about the tables, but those are similar enough in both).

**Comment by mcjaeger** [comment]
ehem, allright, I got this. The point is to do the conversion of table manually.

**Comment by silverhook** [comment]
Ah, OK. Well the Markdown and Textile table markups are similar enough that it shouldn’t be too difficult to move. I can do that, if I learn where to put it :)

**Comment by mcjaeger** [comment]
Hi, I think the safest would be 
at http://archive15.fossology.org/projects/fossology/wiki the section "FOSSology Administrator Documentation"
to https://wiki.fossology.org/ into the corresponding section.
I have started the test thing already with the integration test table overview. so there is no double work so far.

**Comment by silverhook** [comment]
If I understood the [discussion on the mailing list](https://lists.fossology.org/g/fossology/message/3260) and conf call correctly the plan is now to migrate from https://wiki.fossology.org to the GitHub wiki.

I’m moving this to the project board established for that now.

**Comment by shaheemazmalmmd** [comment]
Hello,

This issue has been inactive for several years and appears to no longer be actionable. As part of our issue tracker cleanup, we are closing old inactive issues.

If you believe this is still a valid issue in the current version of FOSSology, please comment with updated information or open a new issue with reproduction steps, and we'll be glad to investigate.

Thank you for your understanding and your support of the project!
